File: DietMenu.WebUi/src/models/Ingredient/Ingredient.ts
import { Guid } from "guid-typescript";

export class Ingredient {
    id: string = Guid.create().toString();
    exchanger: number = 0;
    is_system: boolean = false;
    name: string = "";
    type_id: string = Guid.createEmpty().toString();
    type_name = "";
    unit_symbol = "";
}

File: DietMenu.Core/src/SilentMike.DietMenu.Core.Infrastructure/EntityFramework/Entities/Ingredient.cs
namespace SilentMike.DietMenu.Core.Infrastructure.EntityFramework.Entities;

using System.Diagnostics.CodeAnalysis;

[ExcludeFromCodeCoverage]
internal sealed class Ingredient
{
    public decimal Exchanger { get; set; } = default;
    public Guid FamilyId { get; set; } = Guid.Empty;
    public int Id { get; set; } = default;
    public Guid InternalId { get; set; } = Guid.Empty;
    public bool IsSystem { get; set; } = default;
    public string Name { get; set; } = string.Empty;
    public string Type { get; set; } = string.Empty;
    public string UnitSymbol { get; set; } = string.Empty;
}
